src/app/app.config.ts

```diff
@@ -2,7 +2,8 @@ import { ApplicationConfig } from '@angular/core';
 import { provideRouter } from '@angular/router';
 
 import { routes } from './app.routes';
+import { provideAnimationsAsync } from '@angular/platform-browser/animations/async';
```

Comment:
This seems unecessary since it doesn't have any import or usages on other parts of the system, why is thi lib needed?

Reply:
Removed